app/styles/studio.css

```diff
@@ -681,6 +682,42 @@ input[type="checkbox"] {
     height: 13px !important;
 }
 
+label[for$="Pre Script"] {
+    font-weight: bold !important;
+}
+
+label[for$="Post Script"] {
+    margin-top: 18px;
+    font-weight: bold !important;
+}
+
+label[for$="Clean Script"] {
+    margin-top: 18px;
+    font-weight: bold !important;
+}
+
+label[for$="Environment Script"] {
+    margin-top: 18px;
+    font-weight: bold !important;
+}
+
+/* buttons for arguments in selection script modal */
+div[name="Arguments"] button {
+  font-size: 12px;
+}
+
+.width100p {
```

Comment:
Usually class name represent what the element does or the context

Reply:
I agree !